src/Campaigns/resources/admin/components/CampaignDetailsPage/Tabs/Settings.tsx

```diff
@@ -3,6 +3,8 @@ import {useFormContext} from 'react-hook-form';
 import {Currency, Editor, Upload} from '../../Inputs';
 import {GiveCampaignDetails} from '../types';
 import styles from '../CampaignDetailsPage.module.scss';
+import {ToggleControl} from '@wordpress/components';
+import campaignPage from './images/campaign-page.svg';
```

Comment:
This is probably nitpicking, but let's change this variable name to something meaningful, like `campaignPageImage` or something similar. `campaignPage` is a bit vague.

Reply:
Resolved in e7ebd6cf2466f806d872e9a82b8bce1d415275b9